content/posts/2018-03-18-dependency-upgrade.md

```diff
@@ -0,0 +1,145 @@
+---
+title: "The Art of a Dependency Upgrade"
+description: "A walk to a technical marvel without breaking a leg or two."
+date: 2018-03-18
+slug: art-of-a-dependency-upgrade
+---
+
+[RxJava](https://github.com/ReactiveX/RxJava) 1.+ reaches EOL on March 31, 2018,
+meaning no further development. That’s not surprising since the 1.+ was
+in bugfix-only mode since June 1, 2017.
+
+This is an interesting event in a software lifetime, since not so many libraries
+actually live and prosper long enough to produce a superior version, at the same time handling
+support for an older version for so long. Fortunately enough, RxJava is one of these lucky projects
+with maintainers actually caring about users.
+Thank you, RxJava maintainers, you are [real human beings and real heroes](https://www.youtube.com/watch?v=-DSVDcw6iW8).
+
+RxJava 1.+ → 2.+ serves as a good example of a _major_ dependency upgrade.
+Developers actually update dependencies pretty frequently, but that’s mostly the case with _minor_ upgrades,
+when things are (usually) nice, cozy and feel like a walk in a park.
+Major upgrades can change API drastically. Another example of such an upgrade
```

Comment:
Move "Another example" to separate paragraph for better readability?

Reply:
Sure, thanks!